pkg/client/openshift/buildconfig.go

```diff
@@ -101,6 +111,37 @@ func (b *buildConfig) TriggerBuild(bc *buildv1.BuildConfig, triggedBy string) (b
 	return true, nil
 }
 
+// TriggerBuildFromFile wil be called by kogito-cli when a build from file is performed.
+// When called a new build will be triggered with the request kogito resource or a tgz file.
+func (b *buildConfig) TriggerBuildFromFile(namespace string, bodyPost io.Reader, options *buildv1.BinaryBuildRequestOptions) (*buildv1.Build, error) {
+
+	result := &buildv1.Build{}
+	buildName := fmt.Sprintf("%s%s", options.Name, "-builder")
```

Comment:
shouldn't we be waiting for the `-binary` buildconfig ?

Reply:
No, binary builds does not expect any s2i build.
In this case, build form asset we do expect a s2i build to build the kogito app using the kogito archetype.

Reply:
right